receiver/filelogreceiver/README.md

```diff
@@ -180,7 +180,126 @@ Exception in thread 2 "main" java.lang.NullPointerException
 
 ## Offset tracking
 
-`storage` setting allows to define the proper storage extension to be used for storing file offsets. 
+The `storage` setting allows you to define the proper storage extension for storing file offsets.
 While the storage parameter can ensure that log files are consumed accurately, it is possible that
 logs are dropped while moving downstream through other components in the collector.
 For additional resiliency, see [Fault tolerant log collection example](../../examples/fault-tolerant-logs-collection/README.md)
+
+### File storage
```

Comment:
Since this addition essentially documents the `file_storage` extension, wouldn't fit better at https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/extension/storage/filestorage?
Then we could link from here the respective part with something like "from more info about the `file_storage` interanals check <link>".

Also, that would allow us to link from [`Fault tolerant log collection example`](../../examples/fault-tolerant-logs-collection/README.md) as well.

@ycombinator @djaglowski wdyt?

Reply:
I agree. I'd rather not have this receiver get into documentation of other components.

Reply:
This addition documents how `filelog` receiver offsets are stored using the `file_storage` extension.  I think the useful parts of this addition are the details pertaining to the data persisted by the `filelog` receiver (signature, offset, etc.).  So I think this documentation belongs with the `filelog` receiver itself, not the `file_storage` extension.

Reply:
Yeah, I like that separation, @djaglowski.  Thanks for suggesting it. Will do!

Reply:
Documented the information persisted by the `filelog` receiver in 8211b78ea2370622053afb1960333a3bbe78459d.  I'll put up a separate PR for the `file_storage ` extension to describe how to inspect the contents of the bbolt data store.

Reply:
Documented the information persisted by the `filelog` receiver in 8211b78ea2370622053afb1960333a3bbe78459d.  I'll put up a separate PR for the `file_storage ` extension to describe how to inspect the contents of the bbolt data store.

Reply:
> I'll put up a separate PR for the `file_storage ` extension to describe how to inspect the contents of the bbolt data store.

https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/32180